### PR TITLE
X-Content-Type-Options header added (bug 998397)

### PR DESCRIPTION
**Pull request by luckylavish**

### CODE REVIEW

File: requirements/prod.txt [hunk 1]
```diff
@@ -113,6 +113,9 @@ tower==0.4.1
 # urllib3 is required by elasticsearch
 urllib3==1.10.3
 
+# django secure middleware
+django-secure==1.0.1
```
Comment (diox) [inline requirements/prod.txt]: We try to sort the packages in this file by name (alphabetically), can you move this line higher ? Also, no need to add "django secure middleware" comment.

File: mkt/settings.py [hunk 2]
```diff
@@ -204,7 +205,11 @@ def path(*args):
     # a middleware causes a redirect, then other middleware stops processing.
     'mkt.site.middleware.RemoveSlashMiddleware',
     'mkt.site.middleware.CommonMiddleware',
+    'djangosecure.middleware.SecurityMiddleware',
 )
+# prevent the browser from guessing the content type,
+# and force it to always use the type provided in the Content-Type header
```
Comment (chuckharmston) [inline mkt/settings.py]: Comments should be a full sentence that starts with a capital letter and ends with a period.

**Comment by luckylavish** [comment]
@diox Please review

**Comment by diox** [comment]
Looks good. Please add this simple test:

```
diff --git a/mkt/site/tests/test_views.py b/mkt/site/tests/test_views.py
index 9dc9518..7d7c0ad 100644
--- a/mkt/site/tests/test_views.py
+++ b/mkt/site/tests/test_views.py
@@ -199,3 +199,11 @@ class TestCSP(mkt.site.tests.TestCase):
         self.client.post(self.url, json.dumps({'csp-report': {}}),
                          content_type='application/json')
         eq_(log_cef.call_args[0][2]['PATH_INFO'], '/services/csp/report')
+
+
+class TestSecure(mkt.site.tests.TestCase):
+    def test_content_nosniff(self):
+        # Test that django-secure is properly installed and configured
+        # according to our needs.
+        response = self.client.get('/')
+        eq_(response['x-content-type-options'], 'nosniff')
```

**Comment by diox** [comment]
Thanks!

**Pull request merged**